src/main/java/duke/taskList.java

```diff
@@ -0,0 +1,267 @@
+package duke;
+/**
+ * List refers to a task list that is used in the Duke program.
+ * An List object contains the tasks and several methods that work on the task.
+ *
+ * @author Dominic Siew Zhen Yu
+ */
+
+import java.io.*;
+import java.util.ArrayList;
+
+public class taskList {
```

Comment:
As part of convention, your class name should have been capitalised. It should be `TaskList` instead of `taskList`